projects/control-service/projects/model/apidefs/datajob-api/api.yaml

```diff
@@ -269,8 +267,7 @@ paths:
           description: Request a GraphQL-like query.
           schema:
             type: string
-            example: |
-              {
+            example: "{
```

Comment:
It needs to be done this way with the latest open api version. 
Same with all the changes in this file. 
They were required to get the project to build.